metacat-main/src/main/java/com/netflix/metacat/main/api/v1/TagController.java

```diff
@@ -252,7 +253,8 @@ public TagController(
         switch (name.getType()) {
             case CATALOG:
                 //catalog service will throw exception if not found
-                final CatalogDto catalogDto = this.catalogService.get(name);
+                final CatalogDto catalogDto = this.catalogService.get(name, GetCatalogServiceParameters.builder()
```

Comment:
Question (for my learning purpose):
What would happen if we don't explicitly set the params `false`.  Would it fallback to default `false`? In that case, do we need to explicitly set them to `false`. Or we prefer to set values explicitly.

Also, figured in Lombok 1.16.16 they have added `@Builder.Default` to override defaults easily.

Reply:
Good point. By default, the values will be false since in this case the member variables are boolean.  I wanted to make it explicit from every calling method.